src/sklearn_evaluation/plot/matplotlib/bar.py

```diff
@@ -41,9 +44,9 @@ def horizontal(values, labels=None, sort=True, error=None, ax=None):
     y_pos = np.arange(len(values))
 
     if error is None:
-        ax.barh(y_pos, values)
+        ax.barh(y_pos, values, edgecolor="#fff")
     else:
-        ax.barh(y_pos, values, xerr=error)
+        ax.barh(y_pos, values, xerr=error, edgecolor="#fff")
```

Comment:
is there an alternative API that would set this from the `apply_theme` decorator? or do you think it's better to set it here

Reply:
solved with this:
```
plt.rcParams['patch.edgecolor'] = '#fff'
plt.rcParams['patch.force_edgecolor'] = True
```